Source/WebCore/page/Navigator.cpp

```diff
@@ -233,6 +234,25 @@ void Navigator::showShareData(ExceptionOr<ShareDataWithParsedURL&> readData, Ref
     });
 }
 
+#if PLATFORM(COCOA) || ENABLE(PDFJS)
+// https://html.spec.whatwg.org/multipage/system-state.html#pdf-viewing-support
+// Section 8.9.1.6 states that if pdfViewerEnabled is true, we must return a list
+// of exactly five PDF view plugins, in a particular order.
+constexpr size_t dummyPDFPluginCount = 5;
```

Comment:
Any reason we can't call `dummyPDFPluginNames().size()`?

Reply:
No -- I'll change to that.